tests/h/cli/commands/normalize_uris_test.py

```diff
@@ -167,15 +161,13 @@ def test_it_reindexes_changed_annotations(req, index):
     indexer.index.assert_called_once_with(set([annotation_1.id, annotation_2.id]))
 
 
-def test_it_skips_reindexing_unaltered_annotations(req, index):
-    annotation_1 = models.Annotation(userid='luke',
-                                     target_uri='http://example.org/')
-    annotation_2 = models.Annotation(userid='luke',
-                                     _target_uri='http://example.net/',
-                                     _target_uri_normalized='http://example.net')
-
-    req.db.add_all([annotation_1, annotation_2])
-    req.db.flush()
+def test_it_skips_reindexing_unaltered_annotations(req, index, factories, db_session):
+    annotation_1 = factories.Annotation(userid='luke',
+                                        target_uri='http://example.org/')
+    annotation_2 = factories.Annotation(userid='luke',
+                                        _target_uri='http://example.net/')
```

Comment:
Is this correct? In the above examples you changed `_target_uri` to `target_uri`, but not here.